src/Stripe.net/Entities/Accounts/Account.cs

```diff
@@ -6,7 +6,7 @@ namespace Stripe
     using Newtonsoft.Json;
     using Stripe.Infrastructure;
 
-    public class Account : StripeEntity, IHasId, IHasObject
+    public class Account : StripeEntity, IHasId, IHasObject, IPaymentSource
```

Comment:
~Why do we implement `IPaymentSource` on Account?~ Forgot Account is a source now, damn that is confusing lol